simpleio.py

```diff
@@ -101,6 +102,67 @@ def shift_out(dataPin, clock, value, msb_first=True):
             tmpval = bool((value & (1 << i)))
             dataPin.value = tmpval
 
+class Servo:
+    """
+    Easy control for hobby (3-wire) servos
+
+    :param ~pulseio.PWMOut servoPin: PWM pin where the servo is located.
+    :param int servoMin: Minimum amount of microseconds allowed. Varies depending on type of servo.
+    :param int servoMax: Maximum amount of microseconds allowed. Varies depending on type of servo.
+
+    Example for Metro M0 Express:
+
+    .. code-block:: python
+
+        import simpleio
+        import time
+        from board import *
+
+        # attach a servo to pin Digital 10
+        myServo = simpleio.Servo(D10)
+
+        while True:
+            # write an angle of 90deg to the servo object
+            myServo.writeAngle(90)
+            time.sleep(1)
+            # write an angle of 150deg to the servo object
+            myServo.writeAngle(150)
+            # print the location of the servo
+            print(myServo.read())
+            time.sleep(1)
+            # write to the servo in microseconds
+            myServo.writeMicroseconds(5500)
+            time.sleep(1)
+    """
+    def __init__(self, servoPin, servoMin = 0.5, servoMax = 2.5):
+        self.myServo = pulseio.PWMOut(servoPin, frequency = 50)
+        self.servoMin = servoMin
+        self.servoMax = servoMax
+        # prevent weird values due to read before write calls
+        self.angle = 90
+
+    def writeAngle(self, angle):
```

Comment:
Modified to conform to the style guide. Write was being used because it's similar to the [.write()](https://www.arduino.cc/en/Reference/ServoWrite) in arduino